### PR TITLE
[DependencyInjection] Autoconfigurable attributes on methods, properties and parameters

### PR DESCRIPTION
**Pull request by alexandre-daubois**
Fix https://github.com/symfony/symfony-docs/issues/15763

### CODE REVIEW

File: service_container/tags.rst [hunk 1]
```diff
@@ -214,11 +214,65 @@ method::
...
+    // src/Attribute/SensitiveElement.php
+    namespace App\Attribute;
+
+    #[\Attribute(\Attribute::TARGET_CLASS | \Attribute::TARGET_METHOD)]
+    class SensitiveElement
+    {
+        // ...
+    }
+
+We must now update the call to
```
Comment (nicolas-grekas) [inline service_container/tags.rst]: should instead of must?

File: service_container/tags.rst [hunk 2]
```diff
@@ -214,11 +214,65 @@ method::
...
+:method:`Symfony\\Component\\DependencyInjection\\ContainerBuilder::registerAttributeForAutoconfiguration`
+to support ``ReflectionMethod`` and ``ReflectionProperty``::
```
Comment (nicolas-grekas) [inline service_container/tags.rst]: Not ReflectionProperty here IIUC

Comment (alexandre-daubois) [reply service_container/tags.rst]: Artifact from my first draft, that's removed. Thanks!

File: service_container/tags.rst (hunk 1 above)
Comment (alexandre-daubois) [reply service_container/tags.rst]: Yes, better :slightly_smiling_face:

**Review by OskarStark — APPROVED** [review]

**Comment by javiereguiluz** [comment]
Nice one! Alex, thanks a lot for your great contributions!

**Pull request merged**